recipes/faampy/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "faampy" %}
+{% set version = "v0.1.3" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: faampy-{{ version }}.tar.gz
+    url: https://github.com/ncasuk/faampy/archive/{{ version }}.tar.gz
+    sha256: 38590ea676b86222b1fec83bb8291912a84ea2bcedab928761fc89bbafe03a5f
+
+build:
+    skip: True  # [py>35]
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+
+    run:
+        - python
+        - basemap
+        - cartopy
+        - ConfigParser
```

Comment:
`conda`, like `pip`, is case insensitive, but we try to enforce lower case for the sake of simplicity.